libcloud/compute/drivers/outscale.py

```diff
@@ -7948,3 +7948,155 @@ def _to_key_pair(self, key_pair):
             private_key=private_key,
             fingerprint=key_pair["KeypairFingerprint"],
             driver=self)
+
+    def ex_create_api_access_rule(
+        self,
+        description : str = "",
+        ip_ranges : list = list(),
+        ca_ids: list = list(),
+        cns: list = list(),
```

Comment:
Be more precise about the list type and hinting them uder the List[type] format for more compliance with base

Reply:
can you give me an example on how it should look? I'm not familiar with what you are describing

Reply:
`ip_ranges: List[str] = None`

Reply:
ok thx, done!